testslide/runner.py

```diff
@@ -139,7 +139,7 @@ def _print_attrs(self, attrs, *values, **kwargs):
                 "\033[0m\033[{attrs}m{value}\033[0m".format(
                     attrs=attrs, value="".join([str(value) for value in values])
                 ),
-                **kwargs
+                **kwargs,
```

Comment:
You should drop the ",", so [Python 2 tests](https://travis-ci.com/facebookincubator/TestSlide/jobs/227232260) can pass.